bindings/python/iota_sdk/types/network_info.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+from dataclasses import dataclass
+from iota_sdk.types.node_info import NodeInfoProtocol
+
+
+@dataclass
+class NetworkInfo:
+    """Network and PoW related information.
+    """
+
```

Comment:
Total nit, sometimes you have a new line after the description of the class, and sometimes you haven't

Reply:
Should all be solved when we have a format CI 😊